packages/qwik-nx/src/generators/setup-tailwind/lib/update-project.ts

```diff
@@ -0,0 +1,19 @@
+import type { ProjectConfiguration, Tree } from '@nrwl/devkit';
+import { joinPathFragments, updateProjectConfiguration } from '@nrwl/devkit';
+
+import { SetupTailwindOptions } from '../schema';
+
+export function updateProject(
+  tree: Tree,
+  config: ProjectConfiguration,
+  options: SetupTailwindOptions
+) {
+  if (config?.targets?.build?.executor === '@nrwl/webpack:webpack') {
```

Comment:
I'm wondering this will ever be the case, we can only use vite executor

Reply:
@dmitry-stepanenko , removed it.